.travis.yml

```diff
@@ -25,7 +25,7 @@ addons:
 
 matrix:
   include:
-    - python: 3.5
+    - python: 3.6
```

Comment:
Is the beamline py36? Was there a specific reason for this change? (not that I have a problem with it)